### ISSUE TITLE
There is no support for css grid?

### ISSUE THREAD

**Issue by Anenth** [emmetio/emmet-docs#44]

**Comment by natank** [comment]
Still already not

**Comment by mesqueeb** [comment]
Big +1 for css grid. 
### 💯💯💯💯💯
A simple `dg` should become `display grid` in my opinion.

Since VSCode uses emmet by default, not having support for CSS grid is a big productivity hit. 😖

**Comment by vellkan** [comment]
Agree - I'm still using ST3 cause it's missing simple things. And adding own snippets it's impossible.

**Comment by veli-aliev** [comment]
Really? Most popular html/css abbriviation tool and it don't have grid support in 2020? Hope this will be fixed ASAP

**Comment by sergeche** [comment]
Yes, still no one made PR with grid snippets at https://github.com/emmetio/emmet 😔

Отправлено с iPhone

**Comment by minemax-ua** [comment]
How on earth..?

**Comment by rodude123** [comment]
it's 2021 and still no support sad 😞

**Comment by rustedwolf** [comment]
This get's quite annoying - I'm a quite heavy grid user.

**Comment by lesnyrysiek** [comment]
It appears display:grid is supported for 15 months now, but some editors are still using old version.

**Comment by thexpand** [comment]
I can confirm this on the latest version of PhpStorm/WebStorm 2021.3.2. `dg` expands to `display: ruby-text-group;` instead of `display: grid;`

**Comment by Eugeno** [comment]
Greetings from 2024! We're still here.

**Comment by rizemun** [comment]
If you are using webstorm, you can go to settings/preferences, Editor->Live Templates->Zen CSS, there are templates like `d:b` and `d:f`, you can copy `d:b` and in your copy change abbreviation to `d:g` and template text to `display:grid;`.
Or, you can add it from scratch, by defining usage block under template text area to CSS>declaration block
After submit, you will receive `d:g > display:grid` code completion from emmet.
